README.rst

```diff
@@ -237,7 +237,7 @@ or a
 .. _StackOverflow: http://stackoverflow.com/questions/tagged/fipy
 
 .. |GitHub|        image:: https://img.shields.io/github/contributors/usnistgov/fipy.svg
-.. _Github:        https://github.com/usnistgov/fipy
```

Comment:
LeaveCamelCaseAlone!!!

Reply:
Unclear what you mean here. GitHub has more camels than Github, with the added benefit of existing.

Reply:
Apologies. Given the malice you committed with `PySparse` I somehow inverted the order of the diff.